### PR TITLE
Update domains-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
Modified the number of domains that can be added to a subscription.

Reference -

https://learn.microsoft.com/en-us/microsoft-365/admin/m365-feature-descriptions?view=o365-worldwide&tabs=Domains https://learn.microsoft.com/en-us/office365/servicedescriptions/exchange-online-protection-service-description/exchange-online-protection-limits

Per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10163

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/domains-faq.yml [hunk 1]
```diff
@@ -159,7 +159,7 @@ sections:
           
           ::: moniker-end
           
-          Typically, you can add up to 900 domains to your Microsoft 365 subscription.
+          Typically, you can add up to 5000 domains to your Microsoft 365 subscription.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/domains-faq.yml]:
```suggestion
          Typically, you could add up to 5,000 domains to your Microsoft 365 subscription.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by efrene** [comment]
Still waiting for verification from the PM before publishing this update.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**